src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,51 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import React from 'react';
+import { Tab } from '../../type/Tab';
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected : (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props>
+  = ({ tabs, selectedTabId, onTabSelected }) => {
```

Comment:
New line shouldn't start from =
```suggestion
export const Tabs: React.FC<Props> = ({ 
  tabs, 
  selectedTabId, 
  onTabSelected 
}) => {
```